.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
i don't think this file should be committed. perhaps it's good to add it to .gitihnore